core/snippets/snippets/forEachStatement.snippet

```diff
@@ -36,3 +36,10 @@ language: python
 for $1 in $2:
     $0
 ---
+
+language: r
+-
+for (${1:x} in ${2:1:10}) {
```

Comment:
Why `1:10` as a placeholder?

Reply:
That's how r does range(10), so the placeholder as there as a syntax reminder for creating an array. (It's the most common example on cheatsheet sites for r for-loops too, most r coders would recognise it as an array)

Reply:
We talked about this in the community meetup and we don't do this kind of thing in other languages snippets. Unless the default value is actually something that is very commonly used; we don't use them as a instruction for the developer.

Reply:
Or was the issue specifically that the hint placeholder was valid code rather than an explicit word describing what it should be?

Reply:
We discussed it. We had two reasons.
1: Consistency with other language snippets
2: The snippets are not a tutorial on how to code

If you get benefit from it nothing stops you from having these snippets in your own repository. We just don't feel that that this is a route we want to go with community.

@nriley Did I miss something from yesterday session?

Reply:
Nope, think you covered it!

Reply:
Nope, think you covered it!